docs/generators/changelog/script.js

```diff
@@ -0,0 +1,23 @@
+const _fs = require('fs');
+const fs = _fs.promises;
+const main = async () => {
+  try {
+    console.log('started concatenating files');
+    const files = await fs.readdir('./generators/changelog/versions', 'utf8');
+    const filesStream = files
+      .map(file => {
+        console.log(`reading file: ${ file }`);
+        return fs.readFile(`./generators/changelog/versions/${ file }`, 'utf8');
+      })
```

Comment:
instead of reading each file sequentially you can use fs read streams to not block the i/o. 
Check out this [article](https://stateful.com/blog/process-large-files-nodejs-streams) to know how to deal with streams in nodejs and fs.

Reply:
I can be asured that the order will be maintained in this way?